_ap/16/index.adoc

```diff
@@ -13,56 +13,131 @@ tags:
-// However this format and the associated conventions are not sufficient in themselves to ensure
-// that metrics are exposed in a consistent way. Problems include:
-// * Insufficient guidance on metric and label naming to provide consistency and 
-//   establishing the principal of hiding implementation details.
-// * Treatment of this as a first-class API, with established mechanisms for API evolution
-// * etc.
+The metrics API may have multiple endpoints to return metrics in different format.
+For example, the service may provide a `/query` or `/query_range` endpoints for users to query metrics, returning them in a JSON format.
+The service may also provide an endpoint that returns metrics in a Prometheus Text format, the defacto standard format used for scraping metrics in many monitoring systems.
+
+The associated conventions for these formats are not sufficient in themselves to ensure that the metrics are exposed in a consistent way. 
+There is insufficient guidance on metric naming schemes which may lead to lack of consistency, clarity and unintentional exposure of implementation details in the exposed metrics.
 
 ## Applicability
+This applies to any metrics that are exposed via an API to the end user of any managed service.
+
+This does not apply to any managed services which provides access to metrics in other means (e.g. metric visualization via a system such as Grafana)
 
-// Call out when this pattern might apply (and when it should not apply, if relevant)
-// For example:
-//
-// This pattern applies to any service which exposes metrics as Prometheus-scrape endpoints.
-// It does not apply to services which provide access to metrics in other formats, or via
-// means other than an API (e.g. metric visualization via a system such as Grafana).
+This does not have to apply to metrics that are used for internal monitoring systems like dashboards and alerts.
 
 ## Structure
 
-// Describe the pattern.
-//
-// For this metrics example we might link to existing guidlines on naming, augment them with extra rules 
-// which apply to services, explain how the API would allow for graceful API evolution without "flag days".
+### Metric Name
+The metric name should adhere to the format below:
 
-## Participants
+*<service-name>_<optional:sub-component>_<descriptor>_<optional:base-unit>_<optional:type|aggregate>*
 
-// What components and/or teams are directly involved
-//
-// For this metrics example this would include the customer and the control plane via which
-// the metrics are exposed
+* *service-name*: The metric name should be prefixed with the service name (e.g. `rhosak`)
+* *sub-component* (optional): The sub-component of the service that the metric measures.
+This should be used for logically categorizing the metrics. 
+The different sub-components of the service must be agreed on by the team (e.g. `server`, `producer`, `consumer`).
```

Comment:
The example sub-components are based on the Kafka metric categories. Will change these to the sub-components that we agree on for rhosak.